src/components/DebugMenu.js

```diff
@@ -223,4 +222,37 @@ function routeToRow(key, onSelect){
   );
 }
 
+const styles = StyleSheet.create({
+  container: {
+    paddingLeft: "40px",
```

Comment:
Dimensions in RN are unitless, should be `paddingLeft: 40`, same for the ones below, otherwise it crashes(for me at least)

Reply:
My bad. I am testing on the emulator now, so I should be able to catch these easier :)